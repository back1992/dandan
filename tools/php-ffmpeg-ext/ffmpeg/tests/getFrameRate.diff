001+ Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameRate.php on line 2
001- ffmpeg getFrameRate(): 24.00