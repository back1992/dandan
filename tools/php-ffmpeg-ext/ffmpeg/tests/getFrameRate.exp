ffmpeg getFrameRate(): 24.00