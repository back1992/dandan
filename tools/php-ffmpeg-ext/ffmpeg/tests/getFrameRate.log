
---- EXPECTED OUTPUT
ffmpeg getFrameRate(): 24.00
---- ACTUAL OUTPUT
Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameRate.php on line 2
---- FAILED
