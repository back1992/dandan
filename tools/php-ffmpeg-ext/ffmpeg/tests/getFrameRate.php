<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/test.avi');
printf("ffmpeg getFrameRate(): %0.2f\n", $mov->getFrameRate());
?>
