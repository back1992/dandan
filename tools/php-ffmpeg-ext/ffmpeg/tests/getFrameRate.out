Fatal error: Class 'ffmpeg_movie' not found in /var/www/html/dandan/tools/php-ffmpeg-ext/ffmpeg/tests/getFrameRate.php on line 2